cpp/log/cert.h

```diff
@@ -272,6 +277,27 @@ class TbsCertificate {
   DISALLOW_COPY_AND_ASSIGN(TbsCertificate);
 };
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const Cert::Status& status) {
+  switch (status) {
+    case Cert::Status::TRUE:
+      out << "TRUE";
+      return out;
+    case Cert::Status::FALSE:
+      out << "FALSE";
+      return out;
+    case Cert::Status::ERROR:
+      out << "ERROR";
+      return out;
+    case Cert::Status::UNSUPPORTED_ALGORITHM:
+      out << "UNSUPPORTED_ALGORITHM";
+      return out;
+  }
+
+  out << "<unknown Cert::Status: " << static_cast<int>(status) << ")";
```

Comment:
A normal `enum` *is* an `int` (so you can assign to `int`, or from `int`, *without any cast*). The "innovation" with `enum class` (new in C++11) is that you now need to do something explicitly to get the conversion (the cast).